packages/rest/src/__tests__/unit/ajv-factory.provider.unit.ts

```diff
@@ -127,27 +128,27 @@ describe('Ajv factory', () => {
   it('honors request body parser options', async () => {
     ctx
       .bind(RestBindings.REQUEST_BODY_PARSER_OPTIONS)
-      .to({validation: {unknownFormats: ['gmail']}});
+      .to({validation: {formats: {gmail: val => val.endsWith('@gmail.com')}}});
```

Comment:
Do you think we should add this in the docs ?

Reply:
Good idea. See https://ajv.js.org/v6-to-v8-migration.html#removed-options.